tests/lua/uarch-riscv-tests.lua

```diff
@@ -425,32 +423,28 @@ local function write_log_to_file(log, out, indent, last)
     out:write("\n")
 end
 
-local function create_catalog_json_log(contexts)
-    local out = create_json_log_file("catalog")
-    util.indentout(out, 0, "[\n")
-    local n = #contexts
-    for i, ctx in ipairs(contexts) do
-        local logFilename = make_json_log_file_name(ctx.test_name, "-steps")
```

Comment:
This change dropped the `-steps` suffix in the `logFilename` field of `uarch-reset` and `send-cmio-response`.
Diff 
``` 
--- /tmp/old/catalog.json	2024-08-09 10:40:07.000000000 -0300
+++ /tmp/new/catalog.json	2024-09-04 08:53:12.209167315 -0300

-  {"logFilename": "uarch-reset-steps.json", "binaryFilename": "", "steps": 1, "proofs":true, "proofsFrequency":1, "initialRootHash": "85d625da70a4582fd2085c074fb8dc8538fdcb9caafa67f5ddd5272d74223d90", "finalRootHash": "5b50373e88c3c73ad12e6585b5501aec912dbc5c487760c55068bcdd47038c76"},
-  {"logFilename": "send-cmio-response-steps.json", "binaryFilename": "", "steps": 1, "proofs":true, "proofsFrequency":1, "initialRootHash": "6ea144ae08d02645b0412b2b76a56dc74fe59fd89031967dbb28615731b98f4d", "finalRootHash": "0314add8d77e819e6866f7f66e6d782915628fae569d333cdc76d8071a52ed96"}
+  {"logFilename": "uarch-reset", "binaryFilename": "", "steps": 1, "proofs":true, "proofsFrequency":1, "initialRootHash": "85d625da70a4582fd2085c074fb8dc8538fdcb9caafa67f5ddd5272d74223d90", "finalRootHash": "5b50373e88c3c73ad12e6585b5501aec912dbc5c487760c55068bcdd47038c76"},
+  {"logFilename": "send-cmio-response", "binaryFilename": "", "steps": 1, "proofs":true, "proofsFrequency":1, "initialRootHash": "6ea144ae08d02645b0412b2b76a56dc74fe59fd89031967dbb28615731b98f4d", "finalRootHash": "0314add8d77e819e6866f7f66e6d782915628fae569d333cdc76d8071a52ed96"}
 
 ```